### PR TITLE
refactor: standardize registry links

### PR DESCRIPTION
**Pull request by jniles**
This refactor updates the registries to have links to their receipts inline, as in the patient card.  Every registry has been updated in an effort to standardize their user experience.

Additionally, the patient links have been removed in favor of the drop down menu.

Finally, the other registries filter views are updated so that a full UUID is never shown to the client.

----

Thank you for contributing!

Before submitting this pull request, please verify that you have:
 - [x] Run your code through ESLint.  [Check out our styleguide](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/docs/STYLEGUIDE.md).
 - [x] Run integration tests.
 - [x] Run end-to-end tests.
 - [x] Accurately described the changes your are making in this pull request.

For a more detailed checklist, [see the official review checklist](https://docs.google.com/document/d/1nupLVLRXgSZJQo_acLgrwvPnN8RukfSiwRhSToj81uU/pub) that this PR will be evaluated against.

Ensuring that the above checkboxes are completed will help speed the review process and help build a stronger application.  Thanks!

### CODE REVIEW

File: client/src/js/services/CashService.js [hunk 1]
```diff
@@ -142,15 +142,21 @@ function CashService(Modal, Api, Exchange, Session, moment, $translate) {
       { field : 'defaultPeriod', displayName : 'TABLE.COLUMNS.PERIOD', ngFilter : 'translate' },
       { field : 'invoiceReference', displayName: 'FORM.LABELS.INVOICE'},
       { field : 'patientReference', displayName: 'FORM.LABELS.REFERENCE_PATIENT'},
-      { field : 'invoice_uuid', displayName: 'FORM.LABELS.INVOICE'}
+      { field : 'invoice_uuid', displayName: 'FORM.LABELS.INVOICE' },
     ];
 
     // returns columns from filters
     return columns.filter(function (column) {
+      var LIMIT_UUID_LENGTH = 6;
```
Comment (sfount) [inline client/src/js/services/CashService.js]: :+1: This is a good change to make for scenarios where we need to display a UUID. In the upcoming refactor on filters we should be able to display the patient's name to get around this bad UX.

**Review by sfount — COMMENTED** [review]
Changes LGTM. One of our upcoming development focusses should be to make sure that all registries behave in a predictable way. 

As this pull request links documents through the reference link template, should this remove the double click to view document behaviour?

**Comment by jniles** [comment]
@sfount, changes made.

**Comment by jniles** [comment]
@kwilu r+

**Comment by ghost** [comment]
:pushpin: Commit f9f32f6 has been approved by `jniles`

**Pull request merged**